### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#69] [Announcements]
# about

粥里有勺糖的博客主题，基于 vitepress 实现

http://localhost:5174/about.html

**Comment by Jack-0105** [comment]
heheh

**Comment by Tricker39** [comment]
可以问一下，图片预览怎么实现的吗？翻看你的源码好像只是引入了 element-plus ，没找到具体配置的地方，方便回复一下，万分感谢

**Reply by ATQQ** [reply]
源码在这里
组件：https://github.com/ATQQ/sugar-blog/blob/master/packages/theme/src/components/BlogImagePreview.vue
引用：https://github.com/ATQQ/sugar-blog/blob/edb66e9f3c7175389b3caec26705597f5adf8687/packages/theme/src/components/BlogApp.vue#L26-L33

**Comment by HyjoyX** [comment]
大佬，公告插件可以控制在哪些页面展示，哪些也页面不可以展示呢，我通过watch监听当前页面是否展示layout:'home',手动添加去除theme-blog-popover类的显隐，效果不太理想，https://github.com/AutumnFishs/blog/blob/main/docs/.vitepress/components/MyLayout.vue

**Reply by ATQQ** [reply]
可以呀。你可以看看readme 有提供配置控制

**Reply by ATQQ** [reply]
https://github.com/ATQQ/sugar-blog/blob/master/packages%2Fvitepress-plugin-announcement%2FREADME.md

你看有个onRouterChange配置 目前可以通过这个配置

**Reply by HyjoyX** [reply]
okok，谢谢

**Comment by yuntizhinan** [comment]
你好，我是速界机场专线的机场主，可以申请一下收录吗？Telegram：@lsq_168 链接：https://speedworld.cc/

**Reply by ATQQ** [reply]
这不太合适吧 hh
